azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/CallingViewModel.kt

```diff
@@ -30,6 +31,7 @@ internal class CallingViewModel(
     private val callScreenOptions: CallCompositeCallScreenOptions? = null,
     val multitaskingEnabled: Boolean,
     val avMode: CallCompositeAudioVideoMode,
+    private val callType: CallType? = null,
```

Comment:
Possible for a non-null default here? I.e. CallType.Undefined.

Reply:
Hi Adam, We do not have call type undefined. This should never be null, As on launch call type is set. However as config object is somehow nullable(should not be), thus it forces to set type null. I think I will create a KTLO to remove null from call config as it can never be null.